### DISCUSSION TITLE
aggregate 表 bitmap_union 列insert的时候可以不填写数据吗？

### DISCUSSION THREAD

**Discussion by qiaohanqing** [apache/doris#21689] [A - General / Q&A]
aggregate 表 bitmap_union 列insert的时候可以不填写数据吗？
	`card_no` varchar(30) not null COMMENT "会员卡号",
	`mobile` varchar(11) replace_if_not_null COMMENT "手机号",
	`aios_openid` varchar(30) replace_if_not_null comment "aios微信小程序id",
	`aios_unionid` varchar(30) replace_if_not_null comment "aios微信unionid",
	`deliver_openid` varchar(30)	replace_if_not_null COMMENT "deliver微信小程序id",
	`deliver_unionid` varchar(30)	replace_if_not_null COMMENT "deliver微信unionid",
	`lece_openid` varchar(30) replace_if_not_null COMMENT "lece微信小程序id",
	`member_openid` varchar(30) replace_if_not_null COMMENT "会员小程序id",
	`unionid` varchar(30) replace_if_not_null COMMENT "member和lece微信unionid",
	`userid` varchar(50) replace_if_not_null COMMENT "支付宝id",
	`is_member` smallint replace_if_not_null COMMENT "是否是会员1：是，0：否",
    `userids` bitmap BITMAP_UNION null COMMENT "使用过的支付宝id"
)ENGINE=OLAP 
AGGREGATE key(`card_no`)
... ... 

我想插入一条数据不带`userids`这个bitmap列如下，就报错  
INSERT into ads.ads_oneid(card_no,unionid)VALUES(111,"222")
> 1105 - errCode = 2, detailMessage =  object-stored column userids mush in insert into columns

请问有什么好的办法解决吗

**Comment by yagagagaga** [comment]
You can try this sql:
```sql
insert into ads.ads_oneid(card_no, unionid,userids) values(111, "222",bitmap_empty());
```
